src/test/java/org/assertj/core/api/Assertions_assertThat_with_Throwable_Test.java

```diff
@@ -111,6 +111,15 @@ public void should_fail_with_good_message_when_assertion_is_failing() {
                                                    .withMessageContaining("<\"bam\">").withMessageContaining("but was:")
                                                    .withMessageContaining("<\"boom\">");
   }
+  
+  @Test
+  public void should_fail_with_good_message_when_vararg_has_message_containing_assertion_is_failing() {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThatThrownBy(raisingException("boom")).hasMessageContaining("bam", "boom"))
+                                                   .withMessageContaining("Expecting:")
+                                                   .withMessageContaining("<\"boom\">")
+                                                   .withMessageContaining("<[\"bam\", \"boom\"]>").withMessageContaining("but could not find:")
+                                                   .withMessageContaining("<[\"bam\"]>");
```

Comment:
let's avoid repeating call to `withMessageContaining` ;-)
for readability put each expected string on a new line (expect for the first).

Reply:
and for consistency, let's update the other tests in this file the same way.